test/brave-validation-skip.test.ts

```
@@ -0,0 +1,145 @@
+import { describe, it, expect, afterEach } from "vitest";
+
+const repoRoot = path.resolve(__dirname, "..");
+
+describe("configureWebSearch non-interactive Brave validation failure", () => {
+  const tmpFiles: string[] = [];
+
+  afterEach(() => {
+    for (const f of tmpFiles) {
+      try {
+        fs.unlinkSync(f);
+      } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)